recipes/kealib/all/patches/fix-export-symbols-and-cmake.patch

```diff
@@ -0,0 +1,216 @@
+--- a/CMakeLists.txt
```

Comment:
@SpaceIm 
Have you sent this patch upstream?

Reply:
Not yet. I've opened an issue: https://github.com/ubarsc/kealib/issues/7, and commented first draft from library author https://github.com/ubarsc/kealib/compare/exportheader
But yes I should open a PR with this patch.

Reply:
https://github.com/ubarsc/kealib/pull/10